### DISCUSSION TITLE
Disabling meshing of the linkerd containers by default

### DISCUSSION THREAD

**Discussion by listenerme** [linkerd/linkerd2#4617] [General]
LInkerd containers are automatically meshed after the deployment of linkerd. How can i disable that behavior? I want to mesh only application containers, not the linkerd containers

**Comment by grampelberg** [comment]
Why do you want to exclude the control plane from the mesh?

**Comment by listenerme** [comment]
Because, sometimes I am getting "linkerd-proxy not ready" while using "linkerd check" and "linkerd stat"

I am creating the namespace "linkerd" separately and annotating it with admission-webhooks to disabled before deploying linkerd

**Reply by grampelberg** [reply]
Sounds like you've got more fundamental issues. The proxy is (by design) required in the control plane.

**Reply by listenerme** [reply]
ok, understood
